00-HTML-CSS-basics/Punto 2/2/style.css

```diff
@@ -0,0 +1,96 @@
+!html {
+	font-size: 14px;
+	font-family: Arial;
```

Comment:
Remember to use fallback fonts http://www.w3schools.com/cssref/css_websafe_fonts.asp